metricbeat/docs/autodiscover-kubernetes-config.asciidoc

```diff
@@ -0,0 +1,18 @@
+Metricbeat supports templates for modules:
+
+["source","yaml",subs="attributes"]
+-------------------------------------------------------------------------------------
+metricbeat.autodiscover:
+  providers:
+    - type: kubernetes
+      templates:
+        - condition:
+            contains:
+              kubernetes.labels.project: "prometheus"
```

Comment:
This is a very good example for the configuration! Perhaps we should use the `prometheus.io.scrape: "true"` annotation?